ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/DefaultOutputCustomizer.java

```diff
@@ -185,32 +168,73 @@ private boolean endsWith(Path path, Path relativePath) {
         return true;
     }
 
-    /*
-     * Selects and shows the specified line of text in editor
+    /**
+     * Finds a file by its path, opens it in editor and sets the text selection and
+     * reveals according to the specified line and column numbers
+     * 
+     * @param file
+     * @param lineNumber
+     * @param columnNumber
      */
-    private void selectRange(EditorPartPresenter editor, int line) {
-        if (editor instanceof TextEditor) {
-            TextPosition startPosition = new TextPosition(line - 1, 0);
-            int lineOffsetStart = ((TextEditor) editor).getDocument().getLineStart(line - 1);
-            if (lineOffsetStart == -1) {
-                lineOffsetStart = 0;
-            }
+    static void openFileInEditorAndReveal(AppContext appContext, EditorAgent editorAgent, Path file,
```

Comment:
I've meant to make it to be package package available this time. Can't imagine a class that may be in need of calling this method other than these two `*OutputCustomizer`-classes at the moment.
If output customizers will be distributed among the different packages in future, we'll have to make it to be like public API. However at the moment, we have all these classes placed closely in one package and we don't have a mean that let adopters to add their own customizers - so no need to make it like public API.
IMHO